monitoring/monitors/viya/podMonitor-sas-go-pods.yaml

```diff
@@ -16,13 +16,18 @@ spec:
     tlsConfig:
       insecureSkipVerify: true
     relabelings:
-    # Enable this when a critical mass of Go services have been rebuilt to provide the
-    # correct value for the sas.com/kustomize-base annotation
-    # - sourceLabels: [__meta_kubernetes_pod_annotation_sas_com_kustomize_base]
-    #   regex: golang
-    #   action: keep
+    - sourceLabels: [__meta_kubernetes_pod_annotationpresent_prometheus_io_port, __meta_kubernetes_pod_annotation_prometheus_io_port]
+      action: replace
+      regex: true;(.+)
+      replacement: $1
+      targetLabel: __port
+    - sourceLabels: [__meta_kubernetes_pod_ip, __port]
+      action: replace
+      regex: (.+);(.+)
+      replacement: $1:$2
+      targetLabel: __address__
     - sourceLabels: [__meta_kubernetes_pod_label_app_kubernetes_io_name]
-      regex: "sas-arke|sas-launcher"
+      regex: "sas-arke|sas-launcher|sas-redis-operator"
```

Comment:
See my comment re: adding the sas-opendistro-exporter pod to the exclusion list.

Reply:
Went ahead and added it to the exclusion list like we discussed.